sdf_timing/tests/data/golden/fixpoint.sdf

```diff
@@ -7,18 +7,18 @@
         (INSTANCE slicem/lut_c)
         (DELAY
             (ABSOLUTE
-                (COND (A == 1'b0 && B == 1'b0 && C == 1'b0)
+                (COND (B == 1'b0 && C == 1'b0 && D == 1'b0)
```

Comment:
any idea why the conditions here change?

Reply:
The original output file was wrong. Compare to the input `fixpoint.sdf`. I do not remember if this was a problem in the parser or in the `sdfwrite.emit()`. If you compare the new golden output file with the test input file, they should match now.